Encounter1/MainActivity.cs

```diff
@@ -2,28 +2,53 @@
 using Android.OS;
 using Android.Runtime;
 using AndroidX.AppCompat.App;
-using Android.Views;
 using Android.Widget;
 using System.IO;
 using System;
 using SQLite;
+using Android.Graphics.Drawables;
+using Xamarin.Forms.Platform.Android;
 
 namespace Encounter1
 {
     [Activity(Label = "EncounterMe", Theme = "@style/AppTheme", MainLauncher = true)]
+    
+    
     public class MainActivity : AppCompatActivity
     {
         EditText txtUsername;
         EditText txtPassword;
         Button btnCreate;
         Button btnSign;
 
+        private AnimationDrawable animationDrawable;
+        
         protected override void OnCreate(Bundle bundle)
         {
             base.OnCreate(bundle);
             SetContentView(Resource.Layout.Main);
             // Set our view from the "main" layout resource
 
+
+            //LinearLayout layout = (LinearLayout)FindViewById(Resource.Id.linearLayout1);
+           // layout.SetBackgroundResource(Resource.Drawable.background);
+
+            animationDrawable = (Android.Graphics.Drawables.AnimationDrawable)Resources.GetDrawable(Resource.Drawable.background);
+            LinearLayout img = (LinearLayout)FindViewById(Resource.Id.linearLayout1);
+            img.SetBackground(animationDrawable);
+
+            //ImageView img = (ImageView)FindViewById(Resource.Id.linearLayout1);
+            //img.SetImageDrawable((Android.Graphics.Drawables.Drawable)animationDrawable);
```

Comment:
Will these be used in the future?

Reply:
No, we will not need them. I forgot to delete them. Thanks for showing me